www/js/controller.js

```diff
@@ -575,6 +575,12 @@ module.exports = {
     global.platform.ipc.send('show-empty-slide');
     global.platform.ipc.send('show-text', { text, isGurmukhi, isAnnouncement });
   },
+  sendMimicShabad(mimicShabad) {
```

Comment:
`menu.js` also should call this in the `dhanGuruSlideButton` click handler.

Reply:
What does that mean?

Reply:
Like it should have an event?